src/main/java/frc/robot/subsystems/ball/shooter/ShooterImpl.java

```diff
@@ -42,32 +47,46 @@ private void setShooterSpeed(double speed) {
         this.shooterController2.set(speed);
     }
 
-    public void invertHatch() {
-        if(this.hatch.get() == DoubleSolenoid.Value.kReverse) {
+    public boolean getHatch() {
+        if(this.hatch.get() == DoubleSolenoid.Value.kForward) {
+            return true;
+        } else {
+            return false;
+        }
+    }
+
+    public void setHatch(boolean position) {
+        if(position == true){
```

Comment:
`if (position)` would suffice

Reply:
The funniest thing about that is I literally did it that way somewhere else like a minute ago